discovery/gossiper.go

```diff
@@ -1616,17 +1648,25 @@ func (d *AuthenticatedGossiper) processNetworkAnnouncement(
 		// newer update for this node so we can skip validating
 		// signatures if not required.
 		if d.cfg.Router.IsStaleNode(msg.NodeID, timestamp) {
+			log.Debugf("Skipped processing stale node: %x",
+				msg.NodeID)
 			nMsg.err <- nil
 			return nil, true
 		}
 
 		if err := d.addNode(msg, schedulerOp...); err != nil {
-			if routing.IsError(err, routing.ErrOutdated,
-				routing.ErrIgnored) {
+			log.Debugf("Adding node: %x got error: %x",
+				msg.NodeID, err)
 
-				log.Debug(err)
-			} else if err != routing.ErrVBarrierShuttingDown {
+			switch {
+			case routing.IsError(err, routing.ErrOutdated,
+				routing.ErrIgnored):
+
+			case err == routing.ErrVBarrierShuttingDown:
+
+			default:
 				log.Error(err)
```

Comment:
Can get rid of this now?

Reply:
hmmm I think we need to log this error when it's not one of the three?